Misc/NEWS.d/next/Windows/2018-05-16-11-31-17.bpo-29097.9mqEuI.rst

```diff
@@ -0,0 +1,2 @@
+Fix bug where ``datetime.fromtimestamp`` erronously throws an OSError on
```

Comment:
It would be better if we can link to the `fromtimestamp()` documentation:

```rst
:meth:`datetime.fromtimestamp`
```

Reply:
OSError -> :exc:\`OSError\`

Reply:
Oh cool, didn't know we could use documentation syntax here.